front/lib/analytics/feedback.ts

```diff
@@ -0,0 +1,38 @@
+import type { estypes } from "@elastic/elasticsearch";
+
+import type { ElasticsearchError } from "@app/lib/api/elasticsearch";
+import { ANALYTICS_ALIAS_NAME, withEs } from "@app/lib/api/elasticsearch";
+import type { Authenticator } from "@app/lib/auth";
+import type { WorkspaceType } from "@app/types";
+import type { AgentMessageAnalyticsFeedback } from "@app/types/assistant/analytics";
+import type { AgentMessageType } from "@app/types/assistant/conversation";
+import type { Result } from "@app/types/shared/result";
+
+function constructDocumentId(
```

Comment:
`nit` to stick with existing pattern
```suggestion
function makeDocumentId(
```